Jenkinsfile

```diff
@@ -32,13 +32,18 @@ node('rhel7'){
         packageJson.extensionDependencies = ["ms-kubernetes-tools.vscode-kubernetes-tools"]
         writeJSON file: 'package.json', json: packageJson, pretty: 4
         sh "vsce package -o openshift-connector-${packageJson.version}-${env.BUILD_NUMBER}.vsix"
+		def findVsix = findFiles(glob: '**.vsix')
+		sh "sha256sum ${findVsix[0].path} > openshift-connector-${packageJson.version}-${env.BUILD_NUMBER}.vsix.sha256"
 	}
 
 	if(params.UPLOAD_LOCATION) {
 		stage('Snapshot') {
 			def filesToPush = findFiles(glob: '**.vsix')
```

Comment:
Well, your pr would definitely work, but implementation contains code duplication or CD.

Try to change glob pattern to include both vsix and sha256 files and that will fix the issue without CD.